### PR TITLE
#167 comments on posts

### PR DESCRIPTION
**Pull request by OussamaJmili**
#167 #168 #169 #164

* User can now comment on a post
* User can edit/delete his comment
* Comments are visible to all users with the name of the commenter
* fixed some problems with edit/delete post
* made the page details responsive

### CODE REVIEW

File: linkedin-clone/src/components/PageDetails.js [hunk 1]
```diff
@@ -150,15 +287,21 @@ const EditPostButton = ({ onClick }) => {
 };
 
 const DeletePostButton = ({ onClick }) => {
-  const handleClick = () => {
+  const handleClick = async () => {
     if (window.confirm("Are you sure you want to delete this post? This action cannot be undone.")) {
```
Comment (drbladz) [inline linkedin-clone/src/components/PageDetails.js:291]: Nice use of confirmation prompt. Can you add them for delete group and leave group?

**Comment by MomooDiane** [comment]
It is nice that you have enabled comments on a post! Makes the company pages more interactive

**Review by MomooDiane — APPROVED** [review]

File: linkedin-clone/src/components/PageDetails.js (hunk 1 above)
Comment (OussamaJmili) [reply linkedin-clone/src/components/PageDetails.js:291]: sure, approve this PR and I'll create a new branch/PR for it

**Review by drbladz — APPROVED** [review]
Nice comments feature

**Pull request merged**